src/components/images/ImagePreviewOverlay.tsx

```diff
@@ -34,7 +35,40 @@ import {
   Cover,
   useColorMode,
 } from '@rainbow-me/design-system';
-import { usePersistentAspectRatio } from '@rainbow-me/hooks';
+import { useDimensions, usePersistentAspectRatio } from '@rainbow-me/hooks';
+import { ImgixImage } from '@rainbow-me/images';
+import styled from '@rainbow-me/styled-components';
```

Comment:
Ideally should use design system instead of `styled-components`. We can use the `as` prop on `Box` to support custom components.

E.g.

```tsx
<Box as={BlurView} blurAmount={100} blurType="light" style={position.coverAsObject}>
```

Reply:
will fix